pkg/grpc/tracing/tracing.go

```diff
@@ -59,6 +60,14 @@ func GetProcessKprobe(event *MsgGenericKprobeUnix) *tetragon.ProcessKprobe {
 			Pid:       &wrapperspb.UInt32Value{Value: event.ProcessKey.Pid},
 			StartTime: ktime.ToProto(event.ProcessKey.Ktime),
 		}
+		go func() {
```

Comment:
Why do we need a goroutine here? Why not just call `procevents.RetryRunningProcByPid(event.ProcessKey.Pid)`?

Reply:
```go
                 tetragonProcess = &tetragon.Process{
			Pid:       &wrapperspb.UInt32Value{Value: event.ProcessKey.Pid},
			StartTime: ktime.ToProto(event.ProcessKey.Ktime),
		}
```
considering that a process information has been mocked, and `RetryRunningProcByPid` cannot return a tetragon.Process immediately, so i used goroutine. Wouldn't it be better to return a tetragon.Process immediately?

Reply:
thinks, maybe it's better not to use goroutine, which seems to help generate fewer events without full process information even without immediately returning a tetragon.Process